_inc/client/plans/single-product-backup.jsx

```diff
@@ -6,6 +6,35 @@ import { translate as __ } from 'i18n-calypso';
 
 import './single-product-backup.scss';
 
+export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
+	const perYearPriceRange = `${ currencySymbol }${ backupPlanPrices.jetpack_backup_daily.yearly }-${ backupPlanPrices.jetpack_backup_realtime.yearly } /year`;
+
+	const full_daily_backup_yearly_cost = backupPlanPrices.jetpack_backup_daily.monthly * 12;
+	const full_realtime_backup_yearly_cost = backupPlanPrices.jetpack_backup_realtime.monthly * 12;
+
+	return (
+		<div className="single-product-backup__plan-price-display-container">
+			<div className="discounted-price__container">
```

Comment:
Since it's the "root" of the CSS block, why are we calling this `discounted-price__container` and not just `discounted-price`?

Also, do we really need two separate CSS blocks for `discounted-price` and `plans-price`? Couldn't we just have a `plan-price` block that would accept a modifier like `is-discounted`? The modifier would change the color of the text and add the slash (most likely we could add the slash as a pseudoelement like `::before`).

Reply:
I like this idea, but I'd like to do it in another PR after this is merged to make testing easier.

Reply:
Sounds good 👍

Reply:
Created https://github.com/Automattic/jetpack/issues/14051